llvm/lib/Target/PICMid/GISel/PICMidInstructionSelector.cpp

```diff
@@ -122,14 +133,50 @@ bool PICMidInstructionSelector::select(MachineInstr &I) {
   if (selectImpl(I, *CoverageInfo)) {
     return true;
   }
-
   switch (I.getOpcode()) {
+  case PICMid::G_STORE:
+  case PICMid::G_LOAD:
+    return selectLoadStore(I);
   case PICMid::G_SHLE:
   case PICMid::G_LSHRE:
     return selectShift(I);
+  case PICMid::G_FRAME_INDEX:
+    return selectFrameIndex(I);
+  // TODO other instructions
   default:
-    return false;
+    return true;
+  }
+}
+
+bool PICMidInstructionSelector::selectFrameIndex(MachineInstr &I) const {
```

Comment:
IIRC, there‘s also a method in PICMidRegisterInfo which resolves frame indices. Need to follow up on that

Reply:
https://github.com/llvm-pic/llvm-pic/blob/a87b349293a8dc5699bd2492b492370ffd2d5867/llvm/lib/Target/PICMid/PICMidRegisterInfo.cpp#L48